### PR TITLE
update login to login admins

### PR DESCRIPTION
**Pull request by ahmadysalah**
create admin query when the applicant not in the database:

- response error updated,
- response applicant data updated.
- response admin user with admin role in cookies either,

### CODE REVIEW

File: server/controllers/auth/login.js [hunk 1]
```diff
@@ -3,35 +3,61 @@ const { sign } = require('jsonwebtoken');
...
-              if (!result)
-                res.status(400).json({ error: 'wrong Email or password' });
+              if (!result) res.status(400).json(erroResponse);
               else {
                 const userToken = { userId: data.id };
                 const cookie = sign(userToken, process.env.SECRET_KEY);
                 res
                   .cookie('applicant', cookie, { httpOnly: true })
-                  .json({ data });
+                  .json({ status: 'successfully', role: 'applicant', data });
```
Comment (mFlaifel) [inline server/controllers/auth/login.js:37]: you send to front-end all user data from database, maybe it is not secure to return all user data,
image

File: server/controllers/auth/login.js [hunk 2]
```diff
@@ -3,35 +3,61 @@ const { sign } = require('jsonwebtoken');
...
+                  else {
+                    const userToken = { userId: data.id };
+                    const cookie = sign(userToken, process.env.SECRET_KEY);
+                    res
+                      .cookie('admin', cookie, { httpOnly: true })
+                      .json({ status: 'successfully', role: 'admin', data });
+                  }
+                })
+              )
+              .catch(() => res.status(400).json(erroResponse));
```
Comment (mFlaifel) [inline server/controllers/auth/login.js:56]: when i try to login as admin i have an error when i use the data from fake data,
image

Comment (ahmadysalah) [reply server/controllers/auth/login.js:56]: password  :+1: 
 'Must Contain 8 Characters, One Uppercase, One Lowercase, One Number and one special case Character'

File: server/controllers/auth/login.js (hunk 1 above)
Comment (ahmadysalah) [reply server/controllers/auth/login.js:37]: I think that is the purpose of login.

File: server/controllers/auth/login.js [hunk 3]
```diff
@@ -3,35 +3,61 @@ const { sign } = require('jsonwebtoken');
...
+        res.status(400).json({
+          status: 'failed',
+          type: 'type error',
+          errorMessage: 'Check the data you entered',
+        });
       else {
         applicant
           .findOne({ email })
           .then((data) => {
             bcrypt.compare(password, data.password, (err, result) => {
```
Comment (ha77an) [inline server/controllers/auth/login.js:30]: @ahmadysalah you used bcrypt without requiring it in top file

Comment (ahmadysalah) [reply server/controllers/auth/login.js:30]: not you can't, but this file doesn't have changed in rows 1 to 4 look again.

Comment (ha77an) [reply server/controllers/auth/login.js:30]: ok, I didn't see it

**Comment by Asmaathabet** [comment]
@ahmadysalah  please change file name from `adminFackeData.json` to `adminFakeData.json`

**Pull request merged**